bridge/qb/server.lua

```diff
@@ -14,8 +17,9 @@ CreateQbExport('GiveKeys', giveKeys)
 local function removeKeys(source, plate)
     local vehicles = GetVehiclesFromPlate(plate)
     for i = 1, #vehicles do
-        RemoveKeys(source, vehicles[i])
+        RemoveKeys(source, vehicles[i], true)
     end
+    exports.qbx_core:Notify(source, locale('notify.keys_removed'))
```

Comment:
Shouldn't we also check for `success` here?

Reply:
Done. Also fixed an issue where only the first vehicle would get keys, since the success short-circuited subsequent GiveKeys calls.